### ISSUE TITLE
Documentation for Web & Windows is wrong. 

### ISSUE THREAD

**Issue by jaredbaszler** [miguelpruivo/flutter_file_picker#343]
In order to get the extensions filter to work correctly with a combo of Flutter, Dart, Chrome & Windows, I needed to list extensions as such:

```
List<html.File> files = await FilePicker.getMultiFile(
      type: FileType.custom,
      allowedExtensions: ['.pdf', '.txt'],
    );
```

The documentation provided doesn't include the leading period and without it the filter didn't apply itself.

**Comment by miguelpruivo** [comment]
@jaredbaszler you shouldn’t need the dots. If so, then the plugin must be updated to not require it so it will be consistently with mobile.

**Comment by jaredbaszler** [comment]
@miguelpruivo - I updated to versions:

file_picker: ^1.13.0+1 (was previously on version 1.13.0)
file_picker_web: ^1.0.1+2 (was previously on version 1.0.0)

and I'm still getting the same behavior.  

WITH DOTS:
![image](https://user-images.githubusercontent.com/9924084/88674274-3b3e1500-d0af-11ea-87d4-0bfa884474b2.png)
![image](https://user-images.githubusercontent.com/9924084/88674337-4ee97b80-d0af-11ea-9ccc-efed63146e26.png)

WITHOUT DOTS:
![image](https://user-images.githubusercontent.com/9924084/88674417-67f22c80-d0af-11ea-8091-f708b5394eb3.png)
![image](https://user-images.githubusercontent.com/9924084/88674484-7a6c6600-d0af-11ea-820b-9bf46dfc85b2.png)

**Comment by miguelpruivo** [comment]
@jaredbaszler yes, what I'm saying is that there might be a bug with the plugin itself that may need to be addressed. It shouldn't have to require the `.` at all.

I'll look into it as soon as possible.

Thank you.

**Comment by miguelpruivo** [comment]
@jaredbaszler fixed on `file_picker_web: 1.0.2`. Let me know if you still find any issue.

Thank you!
